tt_metal/include/compute_kernel_api/mask.h

```diff
@@ -41,9 +41,10 @@ ALWI void mask_tile_init() {
  * |----------------|----------------------------------------------------------------------------|----------|-------------------------------------------------------|----------|
  * | dst_data_index | The index of the tile in DST REG for the data and result                   | uint32_t | Must be less than the acquired size of DST REG        | True     |
  * | dst_mask_index | The index of the tile in DST REG for the mask                              | uint32_t | Must be less than the acquired size of DST REG        | True     |
+ * | data_format    | The format of the data and mask                                            | DataFormat | Must be a valid data format                         | False    |
```

Comment:
Should be documented that only float16/Float16b works for this op?

Reply:
I updated the documentation to include support details for Float16, Float16_b, and Int32, noting that Int32 is only supported on WH_B0 hardware.